### ISSUE TITLE
It is hard to distinguish “o” and “O” in loading progress

### ISSUE THREAD

**Issue by ykcowrebbaj** [hoytech/vmtouch#29]
It is hard to distinguish o and O in loading progress:
    if (j == pages_per_char) {
      if (curr == pages_per_char) fprintf(out, "O");
      else if (curr == 0) fprintf(out, " ");
      else fprintf(out, "o");

      j = curr = 0;
    }
I think replace ‘o and O’ with other characters is better

**Comment by hoytech** [comment]
Yes I agree, unfortunately I don't really want to change vmtouch's output at this point since I know some people parse its output with scripts and I don't want to break these scripts.

I would be open to a switch that displays this in a better way.

In my [vmprobe](https://vmprobe.com) project I am using unicode characters which gives a bigger range of detail:

    $ vmprobe cache -p /home/doug/prelude.pdf show
      localhost:/home/doug/prelude.pdf
        39153/91939 (152M/359M)
        [████▁   ▁   ▁ ▆██▂  █████████▄      ████    ████████████                        ████    ██]
